test/cmd/images_tests.sh

```diff
@@ -238,7 +238,8 @@ os::test::junit::declare_suite_start "cmd/images${IMAGES_TESTS_POSTFIX:-}/delete
 # test deleting a tag using oc delete
 os::cmd::expect_success_and_text "oc get is perl --template '{{(index .spec.tags 0).name}}'" '5.16'
 os::cmd::expect_success_and_text "oc get is perl --template '{{(index .status.tags 0).tag}}'" 'latest'
-os::cmd::expect_success "oc delete istag/perl:5.16 --context='${cluster_admin_context}'"
+project="$( oc project -q )"
```

Comment:
Why was this using the cluster admin context in the first place? Does deleting an imagestream tag require cluster admin rights?